source/features/linkify-code.tsx

```diff
@@ -34,9 +33,6 @@ function init(): void {
 			for (const element of select.all('span.pl-c', wrappers)) {
 				linkifyIssues(element);
 			}
-
-			// Mark code block as touched
-			wrappers.classList.add(linkifiedURLClass);
 		},
```

Comment:
🤔 I have not tested this but this whole file looks wrong? The selector is literally flipped and the class it's looking for is not added here anymore. Also it should not exclude comments

Reply:
If you drop this line specifically it will just try to parse the whole file every time and add multiple links again like in a recent issue I think